qiskit_rigetti/_qcs_provider.py

```diff
@@ -122,7 +122,7 @@ def _configuration(name: str, num_qubits: int, local: bool, simulator: bool) ->
         backend_name=name,
         backend_version="",
         n_qubits=num_qubits,
-        basis_gates=["u1", "u2", "u3", "cx", "id"],
+        basis_gates=["u1", "u2", "u3", "cx", "id", "cz"],
```

Comment:
Wasn't the request to also include `xy` here?

Reply:
`xy` is not an OpenQASM gate—so adding it won't do anything.